src/utils/byte_parsing.ts

```diff
@@ -354,7 +364,7 @@ function isABEqualBytes(buffer : ArrayBuffer, bytes : Uint8Array) : boolean {
 }
 
 export {
-  strToBytes,
+  strToBytes, UTF16StrToBytes,
```

Comment:
It become more difficult to make the difference between `strToBytes` and `UTF16StrToBytes` by name alone.

Also the `UTF16StrToBytes` name is kind of dumb because string are always UTF16 in JS and because we don't even care about that anyway when we stay under string form.
Also, we name it `toBytes` and we get back a typed array of 2 bytes. That's strange.

`videojs-contrib-eme` call it `stringToUint16Array` which makes more sense.

We could add the notion of UTF16 (I would like that better), but I don't know how to declare both the return type and the encoding in the function name. Maybe something like `strToUTF16Array`.

---

Note: I see that there is also a function I seem to have created named `bytesToUTF16Str`. I make the same mistakes as this is not a good name either :p. But let's not talk about this one for now

Reply:
Actually i prefered `strToUTF16Array` but i named it to be coherent with `bytesToUTF16Str`. I renamed it anyway.

Reply:
:+1:

Reply:
> i named it to be coherent with `bytesToUTF16Str`

The [broken windows theory](https://en.wikipedia.org/wiki/Broken_windows_theory) is confirmed :p!
In all seriousness, as we think to both think that the old code was wrong there, it seems better to fix the old behavior than to repeat.